system-addon/lib/TopSitesFeed.jsm

```diff
@@ -283,6 +284,9 @@ this.TopSitesFeed = class TopSitesFeed {
         if (action.data.name === DEFAULT_SITES_PREF) {
           this.refreshDefaults(action.data.value);
         }
+        if (action.data.name === TOP_SITES_PREF && action.data.value) {
+          this.refresh({broadcast: true});
```

Comment:
Oh, that's not what I meant or thought you were talking about. The feed has `browser.newtabpage.activity-stream.feeds.topsites` to control whether it's on or off, so when that gets toggled, it should cause the section to refresh, no?

Reply:
Yes that works locally. Made another push try to make sure https://treeherder.mozilla.org/#/jobs?repo=try&revision=e23c9c961ef7b896594995bba8c0ddcc5be16fa6

This was confusing, the sections seem to use just 1 pref `feeds.section.*` while TopSites have both `feeds.topsites` and `showTopSites`. Do you know if there is intended?